src/features/midi/index.tsx

```diff
@@ -157,6 +160,9 @@ function onMidiMessage(e: WebMidi.MIDIMessageEvent) {
   if (!msg) {
     return
   }
+
+  getRecordedMidiEvents()?.push(msg)
```

Comment:
This is where is added.

Reply:
Got it. I think that'll only work for a plugged in MIDI device instead of also capturing notes played from an ASCII keyboard or by clicking on the piano keys with touch input

Reply:
fixed